### DISCUSSION TITLE
Double-Header Train Entry/Exit with Mixed Elevation Rails

### DISCUSSION THREAD

**Discussion by crackalackintv** [Opinionated-Blueprints/10-Books-Full-of-Rails#74] [Ideas]
As you mentioned, you are looking into mixed elevation rails. I built this to deal with double-headed trains more efficiently. It uses the Exit RM and Entry Underground BP and then goes into the stacker / return.

Feel free to use. 

Does anyone have feedback on this as well?

![20241122154313_1](https://github.com/user-attachments/assets/0dc82da2-4e25-446d-928f-2e6e3fddeda9)

```0eNqtnM1u2zgQgF+l0GFPcsHhPwPscZ9gj0FQKI42FarYgeOkLYq8+0q2I6n20JoZ9RYnzKfhcH5Izsi/ivv2tX7eNZt9cfOraNbbzUtxc/ureGkeN1Xb/25TPdXFTdFu19un7b55q4v3smg2D/WP4gbeS2Touto9blffq8ftZjJWo2NRrEGH7qqmXe2qp+fJSPt+Vxb1Zt/sm/oo+OHDzy+b16f7etdJWH78+8u+Azx+3a96TlEWz9uX7r86EbsndKSVTqYsfnY/BBXeewHOSFpCchjJDKT16+6tfjhwVtV1ku0m/dDs6vXx76AQsP0D4IhwnWTqBpu6l5AAI4XyN7s4WQvG8Z/dkeRT/OzO1IiQ40D+WrX/rR6a3pCrNi/ogE+/wzUCT3wF9FyEBIorJ1zngUQ01FVA/744669Vs7myRDGOS+TPlyhgDzASWVFnBMuTNcEoqz2XFXMecAyvDAN63pTAS5SAuiUEuowxjTJiqMiQapgu6uIgchYUpRUdNU4QR4FgghF1OS3IKTmUEUwwg7KSCeI500kmiKO8ZII4arT2uq3fqn1n73NM4wYmGkZ0PMtHr8/P293+OsmjpCRRP77RUBL14yiQqB9H6UXqRwOYMZfMuUg2YRL2QMYSNxzGhSFDRH2RzcrDv39pq589dZC2wB7pJJaA60eSKzKoILEEHBUXWQIano3IfVCUFbkPjhK5D45a5D4BDfkWcR/SprIz9utgSS7JoCS5JIOS5JIMalEuCWh+slEQakK42IwyQo2VeA0uvZN4TQYl8ZoMSksmiOYvJzl8ZFBWMkEc5SQZcLAedC/vvERpaKh3QaI0HBUlSsNRIrNHw7IXmT2OkpwwPBqevOSEkUEZwRkxKMK1lZckCY+6uZckiQzKk2+YxisB7ylXAj5IJoy6vY+SCeOoJL1Q8m7+tiIoqjpjGtXpztWZMDSUl3fE16KdR6NdkKQIj8aVIEkRGZTIOfArU5Fz4ChJZnBoXAmBf2vtvSJYnMQ1MjJK8oRDw0pUEhTqsBHImhtt3xHOulHiCQ4132jIe1k9uL1TlEvgKPEMh5pzdPzbG6dQkifP1w3ztRd1CYuhcUe5R+Ef9mwTYbkjKX5OnEQRoImsUh8GPWAqTYpu5v0u7IgKZxsPtBhD32iNB91zMrZUSTM802bAHgMbQTGhU+v8JixZunX5wXAdRcuOTrYpR0bV7Hk7ljwe1TW1xGjsxJUvakIOQwtuiy26aUmJq4JBUjOvAlCKvHY6pQwZtTdQwDC4AQ2EupiS3JCNQQivRSnDVTRkZMYVzStDGjVmSwufHUUrkguCUSuEkA/KL1G8yZS5A1PxZgIkKJ582QZjldqQEjaoRDdxZTJSo+RJ8Z+8mGP+tpRTOQBQNeMnmjkegMkXjwCLnNXg/Qdg+JsPE3EUvanG9JWtD7HmsxcA/TxmdMqgcbJnkAOLHBhkxyJHBtmwyIlBBg6Z0VZggLWCjC4DAzyZGXvTPjgyyPTNqQmDzJ6QuyYtCWi4u75LNZQunWmrwlPVtqs2c4tkx5qL6Td7KMxTYZYAC0SYBgIsEmHTbJeFJRpMJzcPM4q/xmPv3fka4ylt2sVwXWBNEFjTk8wQryib7kkzw33zuOq0sd7vmvXqedvWqBH5kY7yLI83prEMz/F4Ws3wPIunk53hBd5e2ptxsS/20n3+p+9msI6G+UA1xEKgbBsmHQ6CkrPBjzdW8XQ2qTxf6oxTeQYLgsALGZ+cNEfQA28WZgSBNwuzgsCbhTlJrEwZM0OtzHpBbM/KGwRxNwtb1DWk8VOmTWL713GR/TslKdz7cTqEkDFppGCozGSegT9C1JYErEcYwSNsYj3CChbDxtxioJtZrFNjfhqBNQ3JRcxwM0h7RCDvfqxngSXubXm2KsmglmWrXgkeYVi26iVObVh25CVObVh25CVObVjL7S3ZVI1lgSWebHh2JPFkzbMjSQuj5tmRxKc1z44kPq1ZdhQkPq1Zyx0kPg2s5Q70Y+twcKOBJZ4MPPVYwSMUTz30VgDgqUfiyYrlZiEIdi+KFfICvXKphmPtWbnBoOBFp1qN1wEi91Q73gToi37qvlJB39ZHENwEjFtuSvsCRE2uUKdrEwPWxIygLL5Ym1ZQpVv8UEd9qPqDD/W0ZsbhAl17wu12XPSagsYP4pHRxeBG1Px1f1wWC/BbwaQkDhlG6LzcCRbJnXkJVvN7RbTGUWaJfIDfnCS7CIoH7uSWvM8HkbJYXvCIweuA4nX0DqJJBR00s4LOaCYKHzsWMISX7FNaUpoHg7+IraSvmg1NQJB5xRsWiZuBLmpPUPgL1soI7EJdJhbHMBOt7KKZ4G9SK7cIir9KrTy/e0Ph71KrsKTpqYfelcX3TuX9l8vc9pU5KPt6GtyVh0/68EmfPrnDJ3f4ZA8j3WmkPYx0p5H2MNKdRrrDSH8a6Q4j/WmkO4z03chOjmZfP/U1suEbe8qire7rtvvdbf+3v8cv07k7/mLyTTx3F0M+/fOj2X/669O/+2r9rdNWWbzVu5djz2J3GLApORtScP2rU/8Dx8ozCw==```

Edit1: Moved support
Edit2: Changed to chain signals on stacker.

**Comment by Bocian-1** [comment]
Implemented! Though I ended up taking this *a bit* further :p

**Reply by crackalackintv** [reply]
Awesome, I'll try those out today!
